### PR TITLE
Added cat/view file (issue #23)

### PR DESCRIPTION
**Pull request by robotamer**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -22,9 +22,9 @@ import (
 
 	"github.com/BurntSushi/toml"
 	"github.com/fatih/color"
-	"github.com/mattn/go-isatty"
-	"github.com/mattn/go-runewidth"
-	"github.com/mattn/go-tty"
+	isatty "github.com/mattn/go-isatty"
+	runewidth "github.com/mattn/go-runewidth"
+	tty "github.com/mattn/go-tty"
```
Comment (mattn) [inline main.go]: please revert this

File: main.go [hunk 2]
```diff
@@ -557,6 +563,50 @@ func cmdEdit(c *cli.Context) error {
...
+	f, err = os.Open(filepath.Join(cfg.MemoDir, files[0]))
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	_, err = io.Copy(os.Stdout, f)
+	return err
+}
+
```
Comment (mattn) [inline main.go]: This does not take second argument.

```
$ memo cat 2017-03-13-foo.md
```
This output content immediately.

Comment (robotamer) [reply main.go]: Yes I know, who is going to remember exact date and name, so I find that to be mute. But you are welcome to add it if you like.

**Comment by mattn** [comment]
Merged. https://github.com/mattn/memo/commits/master

Thank you.

**Pull request closed without merging**